src/utils/isIgnored.ts

```diff
@@ -0,0 +1,34 @@
+import { fileExists, readFile } from '@sasjs/utils'
+import path from 'path'
+import ignore from 'ignore'
+import { getLintConfig, getProjectRoot } from '.'
+import { LintConfig } from '../types'
+
+/**
+ * A function to check if file/folder path matches any pattern from .gitignore or ignoreList (.sasjsLint)
+ *
+ * @param fPath absolute path of file or folder
```

Comment:
shouldn't it be `@param {string} fPath`?

Reply:
fixed